documentation/api/query/v4/catalogs.markdown

```diff
@@ -47,6 +47,7 @@ See [the AST query language page][ast].
   recent catalog.
 * `producer_timestamp` (string): a string representing the time at which the
   `replace_catalog` command for a given catalog was submitted from the master.
+* `producer` (string): the certname of the Puppet master that sends the catalog to PuppetDB.
```

Comment:
@wkalt I'm unclear what change you're suggesting - are you saying everything should be past tense?